docs/usage/shoot_firewall.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Shoot Firewall Configuration
+weight: 15
+---
+
+# Endpoints and Ports of a Shoot Control-Plane
+
+With the [reversed VPN](../operations/reversed-vpn-tunnel.md) tunnel, there are no endpoints with open ports in the shoot cluster required by Gardener.
+In order to allow communication to the shoots control-plane in the seed cluster, there are endpoints shared by multiple shoots of a seed cluster.
```

Comment:
Do you mean that all Shoots on a Seed use the istio ingress-gateway load balancer? Then we should rather rework like "there are endpoints shared by all Shoots running on a Seed cluster".

Reply:
This was copy-pasted from `Endpoints and Ports of a Shoot Control-Plane`